client/client_test.go

```diff
@@ -540,6 +540,47 @@ func TestHandleModifyResponse(t *testing.T) {
 			},
 			ClientError: "received a session parameter result when there was none pending",
 		}},
+	}, {
+		desc: "AckType set to FIB_ACK, response should contains both FIB and RIB",
```

Comment:
Do you think you can also help write some unitest to cover the `AFTResult_RIB_PROGRAMMED` and `AFTResult_FAILED` scenario too (per your changes to client.go)? Thanks :)

Reply:
added